go.mod

```diff
@@ -1,19 +1,21 @@
 module go.mercari.io/yo
 
-go 1.21
+go 1.23.0
```

Comment:
Since memefish v0.4.0 is for Go 1.23, it will be updated accordingly. Additionally, cloud.google.com/go/spanner is expected to release a version for Go 1.23 within two weeks.

github.com/cloudspannerecosystem/memefish v0.4.0